code/modules/admin/topic.dm

```diff
@@ -2016,6 +2016,12 @@
 
 		fax_panel(usr)
 
+	else if(href_list["getplaytimewindow"])
+		if(!check_rights(R_ADMIN))	return
+
+		var/mob/M = locate(href_list["getplaytimewindow"])
+		usr.client.cmd_show_exp_panel(M.client)

```

Comment:
This doesn't sanity check to make sure `M` exists before trying to access its client. Stale topic links will make this runtime.